improver/regrid/landsea.py

```diff
@@ -344,7 +354,7 @@ def process(self, cube, input_land, output_land):
 
             # Store and copy cube ready for the output data
             self.nearest_cube = xyslice
-            self.output_cube = self.nearest_cube.copy()
+            self.output_cube = self.nearest_cube.copy()  # type: ignore
```

Comment:
Is there any reason why you ignored the type here? I'm happy to let mypy return errors for now, and fix them in future work.

Reply:
forgot to remove. done